package.nls.json

```diff
@@ -62,6 +64,7 @@
     "appService.showSavePrompt": "Show warning dialog on remote file uploading.",
     "appService.startStreamingLogs": "Start Streaming Logs",
     "appService.toggleAppSettingVisibility": "Toggle App Value Visibility",
+    "appService.validateServiceConnector": "Validate Service Connector...",
```

Comment:
Since there are no additional prompts, the ellipses should be removed.

<img width="384" alt="image" src="https://github.com/microsoft/vscode-azureappservice/assets/12476526/8b506891-102e-4781-992c-9fc43fe21825">